main.py

```
@@ -517,12 +530,35 @@ def main():
-        for course in course_task:
-            process_course(chaoxing, course, common_config)
-        
-        logger.info("所有课程学习任务已完成")
-        notification.send("chaoxing : 所有课程学习任务已完成")
+        force_repeat = common_config.get("force_repeat_learn", False)
+        repeat_times = common_config.get("repeat_times", 1)
+
+        if force_repeat:
+            # 无限循环：0 或负数
+            if repeat_times <= 0:
+                logger.info("检测到 force_repeat_learn=true 且 repeat_times<=0，进入无限循环模式 (Ctrl+C 停止)")
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)